docs/pages/blog/mui-x-v6.md

```diff
@@ -0,0 +1,188 @@
+---
+title: MUI X v6 is ready for production!
+description: Introducing the new major version of the advanced components.
+date: 2023-03-06T00:00:00.000Z
+authors: ['josefreitas']
+card: true
+tags: ['MUI X', 'News']
+---
+
+<img src="/static/blog/mui-x-v6/card.png" alt="" style="width: 100%; margin-bottom: 16px;" />
+
+[MUI X v6.0.0](https://github.com/mui/mui-x/releases/tag/v6.0.0) is finally here with many improvements, new features, customization abilities, and a more robust foundation to accommodate the changes we want to deliver next.
+
+## ✨ Features highlights ✨
+
+### Data Grid
+
+#### [ApiRef now available for all users](/x/react-data-grid/api-object/)
+
+Manage pagination, scrolling, state, and other attributes through the Data Grid's API object—previously available only in commercial plans, now accessible to all.
+The ApiRef enables developers to implement a whole new range of customizations that rely on programmatic control of the Grid's features.
```

Comment:
We reference it like this in https://deploy-preview-36398--material-ui.netlify.app/x/react-data-grid/api-object/#how-to-use-the-api-object in the docs, should we do the same?

```suggestion
The `apiRef` enables developers to implement a whole new range of customizations that rely on programmatic control of the Grid's features.
```